README.md

```diff
@@ -12,11 +12,23 @@ $ apm install linter-erlc
 ```
 
 ## Settings
-You can configure linter-elixirc by editing ~/.atom/config.cson (choose Open Your Config in Atom menu):
+You can configure linter-erlc by editing ~/.atom/config.cson (choose Open Your Config in Atom menu):
```

Comment:
I would remove this whole "raw" section since you have the description part below.